src/main/java/org/mule/extension/socket/api/connection/tcp/protocol/CustomProtocol.java

```diff
@@ -39,6 +39,10 @@
 
   public CustomProtocol() {}
 
+  public String getClazz() {
+    return clazz;
```

Comment:
Doesn't make sense to enforce this here. DW will never try to instantiate this class